File: src/main/scala/woshilaiceshide/sserver/benchmark/rapidoid/RapidoidHttpServer.java
package woshilaiceshide.sserver.benchmark.rapidoid;

import org.rapidoid.net.*;

public class RapidoidHttpServer {

	public static void main(String[] args) {

		ServerBuilder builder = new ServerBuilder();
		builder.protocol(new SimpleHttpProtocol()).port(8989).workers(2).build().start();
	}

}

File: src/main/scala/woshilaiceshide/sserver/benchmark/rapidoid/RapidoidHttpServer.scala
package woshilaiceshide.sserver.benchmark.rapidoid

import org.rapidoid.net.ServerBuilder

object RapidoidHttpServer extends App with woshilaiceshide.sserver.benchmark.ServerProperty {

  val builder = new ServerBuilder();
  builder.protocol(new SimpleHttpProtocol()).address(interface).port(port).workers(2).build().start();

}